### PR TITLE
Fix Razorbacks not working

### PR DESCRIPTION
**Pull request by naydef**

### CODE REVIEW

File: addons/sourcemod/scripting/freak_fortress_2.sp [hunk 1]
```diff
@@ -4103,10 +4103,10 @@ stock RemovePlayerBack(client, indices[], length)
 stock FindPlayerBack(client, index)
 {
 	new entity=MaxClients+1;
-	while((entity=FindEntityByClassname2(entity, "tf_wearable"))!=-1)
+	while((entity=FindEntityByClassname2(entity, "tf_wearable*"))!=-1)
 	{
 		decl String:netclass[32];
-		if(GetEntityNetClass(entity, netclass, sizeof(netclass)) && StrEqual(netclass, "CTFWearable") && GetEntProp(entity, Prop_Send, "m_iItemDefinitionIndex")==index && GetEntPropEnt(entity, Prop_Send, "m_hOwnerEntity")==client && !GetEntProp(entity, Prop_Send, "m_bDisguiseWearable"))
+		if(GetEntityNetClass(entity, netclass, sizeof(netclass)) && StrContains(netclass, "CTFWearable")>-1 && GetEntProp(entity, Prop_Send, "m_iItemDefinitionIndex")==index && GetEntPropEnt(entity, Prop_Send, "m_hOwnerEntity")==client && !GetEntProp(entity, Prop_Send, "m_bDisguiseWearable"))
```
Comment (winstliu) [inline addons/sourcemod/scripting/freak_fortress_2.sp]: Use `!=` instead

File: addons/sourcemod/scripting/freak_fortress_2.sp [hunk 2]
```diff
@@ -4103,10 +4103,10 @@ stock RemovePlayerBack(client, indices[], length)
 stock FindPlayerBack(client, index)
 {
 	new entity=MaxClients+1;
-	while((entity=FindEntityByClassname2(entity, "tf_wearable"))!=-1)
+	while((entity=FindEntityByClassname2(entity, "tf_wearable*"))!=-1)
```
Comment (winstliu) [inline addons/sourcemod/scripting/freak_fortress_2.sp:4106]: Can you walk me through this?  Is the entity actually named `tf_wearable*` now, or is the `*` the wildcard character?

Comment (naydef) [reply addons/sourcemod/scripting/freak_fortress_2.sp:4106]: wildcard character. The Razorback is now called tf_wearable_razorback, before it was tf_wearable

**Comment by naydef** [comment]
Just to say that this pull request is fully tested on my server: Razorback is working whenever the player takes damage or goomba stomp, razorback does not regenerate itself.

**Pull request merged**